components/navbar/index.tsx

```diff
@@ -5,22 +5,14 @@ import styles from "./navbar.module.scss";
 export default function Navbar({ name, description, avatarSrc }) {
   return (
     <header className={styles.container}>
-      <Link href="/" passHref>
-        <a href="replace by passHref" className={styles.profile}>
+      <Link href="/" className={styles.profile}>
+        <>
           <Avatar src={avatarSrc} alt={name} />
           <div className={styles.profileContent}>
-            <Link href="/" passHref>
-              <a href="replace by passHref" className={styles.name}>
-                {name}
-              </a>
-            </Link>
-            <Link href="/" passHref>
-              <a href="replace by passHref" className={styles.description}>
-                {description}
-              </a>
-            </Link>
+            <p className={styles.name}>{name}</p>
+            <p className={styles.description}>{description}</p>
           </div>
-        </a>
+        </>
```

Comment:
Follow [this](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#link-component)

Replace Link with p tag inside Link component.